ci/cloudbuild/dockerfiles/fedora-latest-cmake.Dockerfile

```diff
@@ -203,14 +195,15 @@ RUN curl -fsSL https://github.com/lvc/abi-dumper/archive/16bb467cd7d343dd3a16782
     mv abi-dumper.pl /usr/local/bin/abi-dumper && \
     chmod +x /usr/local/bin/abi-dumper
 
-# Install the Cloud SDK and some of the emulators. We use the emulators to run
-# integration tests for the client libraries.
-COPY . /var/tmp/ci
-WORKDIR /var/tmp/downloads
-ENV CLOUDSDK_PYTHON=python3
-RUN /var/tmp/ci/install-cloud-sdk.sh
-ENV CLOUD_SDK_LOCATION=/usr/local/google-cloud-sdk
-ENV PATH=${CLOUD_SDK_LOCATION}/bin:${PATH}
+# Install ctcache (with global caching support) to speed up our clang-tidy build
+WORKDIR /var/tmp/build
+RUN curl -fsSL https://github.com/coryan/ctcache/archive/872c14416ea739334c6171761303d07739188dec.tar.gz | \
```

Comment:
should we be using your fork?

Reply:
Until https://github.com/matus-chochlik/ctcache/pull/31 is merged, we have no choice.